opensrp-reveal/src/main/java/org/smartregister/reveal/fragment/DownloadedOfflineMapsFragment.java

```diff
@@ -0,0 +1,122 @@
+package org.smartregister.reveal.fragment;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.Toast;
+
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.adapter.DownloadedOfflineMapAdapter;
+import org.smartregister.reveal.contract.DownloadedOfflineMapsContract;
+import org.smartregister.reveal.model.OfflineMapModel;
+import org.smartregister.reveal.presenter.DownloadedOfflineMapsPresenter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DownloadedOfflineMapsFragment extends Fragment implements DownloadedOfflineMapsContract.View {
+
+    private RecyclerView downloadedMapsrecyclerView;
+    private DownloadedOfflineMapAdapter adapter;
+    private Button btnDeleteMap;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'btnDeleteMap' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)